cmd/podman/kube/play.go

```diff
@@ -182,6 +182,10 @@ func playFlags(cmd *cobra.Command) {
 }
 
 func Play(cmd *cobra.Command, args []string) error {
+	if playOptions.ServiceContainer && !playOptions.StartCLI { // Sanity check to be future proof
+		return fmt.Errorf("--service-container does not work with --start=stop")
```

Comment:
Nit: Should this be part of `docs` as well not sure.

Reply:
`--service-container` is not documented and hidden as it's not meant to be used by users but only in the context of systemd.

Reply:
Got it thanks.